carbonmark/components/pages/Users/SellerConnected/ListingEditable.tsx

```diff
@@ -61,10 +62,13 @@ export const ListingEditable: FC<Props> = (props) => {
     setIsLoading(true);
     try {
       if (!address) return;
+
       const allowance = await getCarbonmarkAllowance({
         tokenAddress: values.tokenAddress,
         userAddress: address,
         network: networkLabel,
+        // @todo add tokenID to project type on the subgraph. temporary workaround only to mock a tokenID to identify as 1155
```

Comment:
Is this still relevant? if so is there a ticket we can link to?

Reply:
I'm not sure there's a clear need for it anymore but it'd definitely be a nice to have for the future. Will create a ticket.